src/styles/variables.scss

```diff
@@ -0,0 +1,11 @@
+$aside-menu-color: #1d1d1d;
+$text-color-dark: #1d1d1d;
```

Comment:
Both variables refer to text color - I am not sure if you need two variables with same value